### PR TITLE
[IMP] Meta files

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,22 +1,46 @@
...
-language: python
+stages:
+  - linting
+  - test
 
-python:
-  - "3.5"
+jobs:
+  include:
+    - stage: linting
```
Comment (moylop260) [inline .travis.yml:22]: You can delete this section since that you don’t have black files configuration

File: .pylintrc [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+load-plugins=pylint_odoo
+score=n
+
+[ODOOLINT]
+readme_template_url="https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst"
+manifest_required_authors=Odoo Community Association (OCA)
+manifest_required_keys=license
+manifest_deprecated_keys=description,active
+license_allowed=AGPL-3,GPL-2,GPL-2 or any later version,GPL-3,GPL-3 or any later version,LGPL-3
+valid_odoo_versions=13.0
```
Comment (sbidoul) [inline .pylintrc:11]:
```suggestion
valid_odoo_versions=12.0
```

File: .pylintrc-mandatory [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+load-plugins=pylint_odoo
+score=n
+
+[ODOOLINT]
+readme_template_url="https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst"
+manifest_required_authors=Odoo Community Association (OCA)
+manifest_required_keys=license
+manifest_deprecated_keys=description,active
+license_allowed=AGPL-3,GPL-2,GPL-2 or any later version,GPL-3,GPL-3 or any later version,LGPL-3
+valid_odoo_versions=13.0
```
Comment (sbidoul) [inline .pylintrc-mandatory:11]:
```suggestion
valid_odoo_versions=12.0
```

File: .travis.yml [hunk 4]
```diff
@@ -1,22 +1,46 @@
-sudo: false
-cache: pip
+language: python
+cache:
+  directories:
+  - $HOME/.cache/pip
+  - $HOME/.cache/pre-commit
+
+python:
+  - "3.6"
```
Comment (sbidoul) [inline .travis.yml:8]:
```suggestion
  - "3.5"
```

**Comment by max3903** [comment]
We'll leave this for version 13.

**Pull request closed without merging**